repository/repository-core/src/main/java/org/eclipse/vorto/repository/core/impl/JcrModelRepository.java

```diff
@@ -101,6 +102,8 @@
 
   private static Logger logger = Logger.getLogger(JcrModelRepository.class);
 
+  private static String VORTO_NODE_TYPE = "vorto:type";
```

Comment:
can be final, as it is not changed , correct ?

Reply:
will fix tomorrow